webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/comment/content/ActivityCommentRichText.vue

```diff
@@ -14,7 +14,7 @@
           v-model="message"
           :ck-editor-type="ckEditorId"
           :max-length="$activityConstants.COMMENT_MAX_LENGTH"
-          :placeholder="$t('activity.composer.placeholder', {0: $activityConstants.COMMENT_MAX_LENGTH})"
+          :placeholder="$t('activity.newComposer.placeholder')"
```

Comment:
It's not new anymore IMO. It would be better to avoid adding `new` inside labels

```suggestion
          :placeholder="$t('activity.composer.placeholder')"
```

Reply:
Done!